services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/component/DccRulesNotFetchedStructureProviderTest.java

```diff
@@ -151,9 +151,9 @@ void shouldContainEmptyInvalidationRules() throws FetchBusinessRulesException {
     businessRuleItem2.setCountry("test2");
 
     BusinessRule businessRule = new BusinessRule();
-    businessRule.setType(RuleType.Acceptance.name());
+    businessRule.setType(RuleType.ACCEPTANCE.name());
```

Comment:
I think here you should use RuleType.ACCEPTANCE.getType().

Reply:
Yep, that's right. Thanks for pointing it out